src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,6 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+
 public class CinemaHallRequestDto {
+
+    @Min(value = 10)
+    @Max(value = 300)
```

Comment:
You can use Size annotation

Reply:
is size works with primitives?

Reply:
![image](https://user-images.githubusercontent.com/110828777/209094422-c3697d54-ff08-402e-885a-1faeb390d69b.png)

![image](https://user-images.githubusercontent.com/110828777/209094446-62e1cc8f-9322-4db9-8e3b-6d89a087a16e.png)

i have problem using @Size

Reply:
but when i use Min and Max

![image](https://user-images.githubusercontent.com/110828777/209094703-1c68fd64-918d-42cd-9ec8-6f94eafed4d4.png)

![image](https://user-images.githubusercontent.com/110828777/209095018-f94461df-6d55-40e9-80a9-f0e958b37646.png)
 
i dont have problems

Reply:
ok 👌